airbyte-integrations/connectors/source-bigquery/integration_tests/acceptance.py

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
I don't think these `/integration_tests` files are needed

Reply:
https://github.com/airbytehq/airbyte/blob/master/buildSrc/src/main/groovy/airbyte-source-acceptance-test.gradle#L22

?

Reply:
@evantahler 
if want to run the tests only locally as we see [here](https://github.com/airbytehq/airbyte/pull/16445), yes.
If you want to run it as part of the CI testing, you have to add it.

Reply:
The `acceptance.py` is indeed required to launch the test suite with the SAT plugin. We should find a better way because it's far from ideal. @DoNotPanicUA could you please create an issue about this?